src/test/java/CreatePlayList.java

```diff
@@ -0,0 +1,136 @@
+import Auto.pages.AutoHomePage;
+import Auto.pages.AutoLoginPage;
+import Auto.pages.AutoPlaylistPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CreatePlayList extends BaseTest1{
+
+    By enterNameList = By.cssSelector("[type ='text]");
+    String playListName = "my list";
+    String firstQ = "\"";
+    String secondQ = ".";
+    String lastQ = "\"";
+    String name= "\""+ playListName+"."+"\"";
+    String tenChars="TenthTenth";
+    String two="two";
+    String elevenChars="elevenChars";
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/211288744-1d274555-5271-4c92-a86c-d663c7786c13.png)

Reply:
I removed all variables and not move to POM and moved locators to Auto.pages.